collections/Laud_Misc/MS_Laud_Misc_280.xml

```diff
@@ -50,93 +45,174 @@
                         <support>parchment</support>
+                        <extent>
+                           iii (17th-century endleaves, modern foliation i–iii) + 129 (foliated iv–vi, 1–126) + ii (17th-century endleaves, foliated 127–128) leaves
+                           <dimensions unit="mm" type="leaf"><height>315</height><width>205</width></dimensions>
+                        </extent>
+                        <collation>
+                           <formula>Quaternions except for fols. 121–124.</formula>
+                           <signatures/>
+                           <catchwords/>
+                           <secFol>me⹎ ipsum</secFol>
+                        </collation>
                      </supportDesc>
+                     <layoutDesc>
+                        <layout columns="2" ruledLines="33">Ruled in drypoint; 2 columns of 
+                           33 lines.
+                        </layout>
+                     </layoutDesc>
                   </objectDesc>
+                  <handDesc hands="3">
+                     <handNote script="protogothic">Protogothic script; at least three hands.</handNote>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote type="flourInit">Opening initial in red (fol. 1r) with simple decoration; both this and the opening rubric highlighted with a pale yellow wash.</decoNote>
+                     <decoNote type="colInit">Coloured initials in red and blue.</decoNote>
+                     <decoNote type="rubrication">Most tituli in red; some written with letters alternating between red and blue.</decoNote>
+                  </decoDesc>
+                  <bindingDesc>
+                     <binding notAfter="1635" notBefore="1633">
+                        <p>Brown tanned calf over laminated pulpboard for Abp. Laud.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
+                     <origPlace>
+                        <country key="place_7000084">German</country>
+                     </origPlace>
                      <origDate calendar="Gregorian" notAfter="1200" notBefore="1190">12th century, end</origDate>
                   </origin>
+                  <provenance from="1633"><persName role="fmo" key="person_54940373">William Laud</persName>, by 1633 (inscribed, fol. 1r).</provenance>
+                  <acquisition when="1635">Part of his first donation to the Bodleian, 1635.</acquisition>
                </history>
                <additional>
                   <adminInfo>
                      <recordHist>
                         <source><ref target="https://medieval.bodleian.ox.ac.uk/about">Summary description</ref> abbreviated from the Quarto Catalogue (H. O. Coxe,  <title>Laudian Manuscripts</title>, Quarto Catalogues II, repr. with corrections, 1969, from the original ed. of 1858-1885). 
```

Comment:
The text in `source` should probably be revised, since the description now goes beyond Coxe...